### ISSUE TITLE
Chrono 7 renames ANCF headers

### ISSUE THREAD

**Issue by minrk** [erdc/proteus#1258]
Chrono 7 has [renamed some ANCF](https://github.com/projectchrono/chrono/blob/7.0.0/CHANGELOG.md#changed-ancf-element-improvements-and-additions) resources, causing compilation to fail with e.g.

```
proteus/mbd/ProtChMoorings.h:13:10: fatal error: chrono/fea/ChElementBeamANCF.h: No such file or directory
   13 | #include "chrono/fea/ChElementBeamANCF.h"
```

Presumably just needs some `#if`s in the right place to include the new names.

**Comment by cekees** [comment]
Thanks @minrk ! I’ll take a look.

**Comment by cekees** [comment]
Fixed by #1261
